app/views/home.html

```diff
@@ -1 +1,15 @@
-<h1>Hi this is home page</h1>
+<h1>Pokédex Tracker</h1>
+
+<div class="hero">
+  <img class="mockup" src="mockup.png" width="700px">
+  <p>A tool for tracking your <a href="http://bulbapedia.bulbagarden.net/wiki/Living_Pok%C3%A9dex" target="_blank" class="link">Living Dex</a> progress. This project's source code is open source, available on Github (<a href="https://github.com/robinjoseph08/api.pokedextracker.com" target="_blank" class="link">API</a> &amp; <a href="https://github.com/robinjoseph08/pokedextracker.com/" target="_blank" class="link">website</a>). Feel free to contribute or leave comments or issues!</p>
```

Comment:
Maybe add another sentence after the first one explaining some use-cases? How you can use it to find the locations of your remaining pokemon, filter by region to keep you focused while collecting (or something), and share it with others to see how you can help each other out?

And just say "This project is open source and you can find the code on Github..." so you don't say source twice

and maybe change the last line to something like "Feel free to report issues, suggest features, or even submit a pull request!"?

copy isn't my strong suite...